srtcore/filelist.maf

```diff
@@ -53,3 +53,9 @@ threadname.h
 utilities.h
 window.h
 
+SOURCES_WIN32_SHARED
```

Comment:
Done - honestly, I was not sure about that part - changed.

Reply:
Yeah, this part isn't actually documented; I needed some solution to keep files in a separate file and I haven't found any good solution for it so I had to invent it :). I plan to add some conditional support for it so that what you wanted to achieve here could be easier specified, without separate variables and adding `PUBLIC_HEADERS_WIN32_SHARED` keywords.